app/jobs/email_reminder_job.rb

```diff
@@ -0,0 +1,56 @@
+class EmailReminderJob < ApplicationJob
```

Comment:
I think we should name this so that it's more clearly associated with in-person proofing. As-is, I think it could be unclear what email reminders this job would be responsible for sending.

Maybe something like `InPersonProofingEmailReminderJob` ? Or, if we can do namespacing here, `app/jobs/in_person_proofing/email_reminder_job.rb` and named as `InPersonProofing::EmailReminderJob` ?